storage/rocksdb/rdb_threads.cc

```diff
@@ -68,7 +68,7 @@ int Rdb_thread::create_thread(
 }
 
 
-void Rdb_thread::signal(bool stop_thread)
+void Rdb_thread::signal(const bool &stop_thread)
```

Comment:
This and other patches in the constification series replaced several pass-by-value args with pass-by-const references, where the arguments are small enough to be passed by value efficiently - like bool here. Has this been done intentionally?